README.md

```diff
@@ -121,6 +121,16 @@ The config file should contain the following:
 
 `conversant` will take care of the rest! As an example, check out [`fortune-teller/config.json`](/conversant/personas/fortune-teller/config.json). When you launch the Streamlit app, the new persona will appear in the drop down menu.
 
+#### Troubleshooting missing personas
```

Comment:
[nit] somewhere in one of the sections above, we should specify that if you wanna run the demo _and modify it_ locally, we strongly recommend forking this repo rather than installing it as a library from PyPI.

Otherwise, editing the package files from the pip-installed version will be a pain for users. 😅

Reply:
And TBH we may wanna repeat that reminder in this section too, since we don't want users code-changing inside a pip installed library.

Reply:
Thanks for the feedback! I've added some clarifying language: https://github.com/cohere-ai/sandbox-conversant-lib/compare/d6b0519...fbf217d

Let me know if you have any further comments. I'll hold off on merging this until I make a couple other outstanding tweaks to the readme and prep to deploy a new version to pypi, such that pypi stays consistent with our main branch.